docs/introduction/templates_placeholders.rst

```diff
@@ -8,13 +8,13 @@ you can make your own HTML templates CMS-ready.
 Templates
 #########
 
-You can use HTML templates to customise the look of your website, define
+You can use HTML templates to customize the look of your website, define
```

Comment:
English has bad spellings and its getting worse as it borrows words from european languages. I just learnt the pronunciation for Fuchsia, Gnocchi and San Jose.... its just terrible.

Anyway, I will undo the s -> z change. Its wasn't wrong before my change, so no need to change it now.